pkg/corerp/frontend/controller/applications/testdata/graph-app-directroute-in.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/frontend",
+        "name": "frontend",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "connections": {
+                "sql": {
+                    "source": "http://backendapp:8080"
```

Comment:
Do we support url source reference without scheme ? e.g. `backendapp:8080` instead of `<scheme>://backendapp:8080` @nithyatsu

Reply:
yes

Reply:
Added the test case.